spec/gelauto_spec.rb

```diff
@@ -23,6 +25,30 @@
     end
   end
 
+  context 'with Sorbet generic types' do
+    it 'uses T.nilable with T.any correctly' do
+      Gelauto.discover do
+        GelautoSpecs::Utility.safe_to_string(nil)
```

Comment:
This definitely works, but I'd like to suggest moving the `Utility` class into spec/support where the other test classes are.

Reply:
Isn't that where it is? Its in The same module as `Response` and `Client`